graph/client.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import print_function, unicode_literals
+import re
+import graph.xml_parser
+import graph.oval_graph
+import webbrowser
+import json
+import argparse
+
+
+class client():
+    def __init__(self, args):
+        self.arg = self.parse_arguments(args)
+        self.source_filename = self.arg.source_filename
+        self.rule_name = self.arg.rule_id
+        self.xml_parser = graph.xml_parser.xml_parser(self.source_filename)
+
+    def run_gui_and_return_answers(self):
+        try:
+            from PyInquirer import style_from_dict, Token, prompt, Separator
+            return prompt(self.get_questions(Separator('= The Rules ID =')))
+        except ImportError:
+            print('== The Rules ID ==')
+            for rule in self.search_rules_id():
+                print(rule['id_rule'] + r'\b')
+            return None
+
+    def get_questions(self, separator):
+        rules = self.search_rules_id()
+        questions = [{
+            'type': 'checkbox',
+            'message': 'Select rule(s)',
+            'name': 'rules',
+            'choices': [
+                separator],
+            'validate': (lambda answer: 'You must choose at least one topping.'
+                         if len(answer) == 0 else True)
```

Comment:
I'm afraid that topping can be found on pancakes or on ice-cream sundae. You probably mean something else.

Anyway, these 2 lines are suspicious. I don't understand it. What is it supposed to do?